ui/app/AppLayouts/Wallet/popups/ReceiveModal.qml

```diff
@@ -135,16 +97,21 @@ StatusModal {
                 flow: Grid.TopToBottom
                 columns: need2Columns ? 2 : 1
                 spacing: 5
+                property var networkProxies: [RootStore.layer1NetworksProxy, RootStore.layer2NetworksProxy]
```

Comment:
we dont need this property right? we can simple assign models as RootStore.layer1NetworksProxy and RootStore.layer2NetworksProxy ?

Reply:
I tried it but the `modelData` was not set in the delegate. Hence being an array it seems just the `index` is set. And in order to access the model by index we need a property (otherwise `model` will be ambiguous in this case).